third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -4123,11 +4123,14 @@ static void fromJsGetMatchedStylesForNode(
   Maybe<protocol::Array<protocol::CSS::RuleMatch>> matchedRules;
   Maybe<protocol::Array<protocol::CSS::PseudoElementMatches>> pseudoIdMatches;
   Maybe<protocol::Array<protocol::CSS::InheritedStyleEntry>> inheritedEntries;
+  Maybe<protocol::Array<protocol::CSS::InheritedPseudoElementMatches>> inherited_pseudo_id_matches;
   Maybe<protocol::Array<protocol::CSS::CSSKeyframesRule>> keyframesRules;
+  Maybe<int> parentLayoutNodeId;
 
   auto response = cssAgent->getMatchedStylesForNode(
```

Comment:
LGTM, but perhaps we should use references instead of pointers to avoid situations where an argument _must_ be supplied.

Reply:
I think this method is defined by upstream chromium, so we don't want to change its interface.